server/logs.lua

```diff
@@ -86,6 +85,7 @@ RegisterNetEvent('qb-log:server:CreateLog', function(name, title, color, message
         logQueue[name][#logQueue[name] + 1] = { webhook = webHook, data = embedData }
 
         if #logQueue[name] >= 10 then
+            local postData = nil
```

Comment:
Try initialising it with our else data, and append content if tag is true (`postData.content = '@everyone'`). This means the else block wouldn't be needed.
`local postData = { username = 'QB Logs', embeds = {} }`